pyupgrade.py

```diff
@@ -1805,8 +1893,8 @@ def _fix_fstrings(contents_text):
     return tokens_to_src(tokens)
 
 
-def fix_file(filename, args):
-    with open(filename, 'rb') as f:
+def fix_file(filename, args):  # type: (Text, argparse.Namespace) -> int
+    with io.open(filename, 'rb') as f:
```

Comment:
hehe, I have another fixer I've been working on that auto-rewrites `io.open(..., 'rb')` => `open(..., 'rb')` so I'd rather rename one of the `f` variables

Reply:
ok